catalog/spatial/registry/registry-federation-admin-service-impl/src/test/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntriesTest.java

```diff
@@ -55,7 +55,7 @@
 import org.mockito.Mock;
 import org.mockito.runners.MockitoJUnitRunner;
 
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(MockitoJUnitRunner.Silent.class)
```

Comment:
It seemed out of the scope to fix here. It could potentially take looking through each when and verify to discover which ones are unstubbed or used incorrectly.